suse_migration_services/units/prepare.py

```diff
@@ -163,12 +177,12 @@ def main():
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
         if os.path.exists(cloud_register_metadata):
-            log.info('Bind mounting /var/lib/cloudregister')
-            Path.create('/var/lib/cloudregister')
+            log.info('Bind mounting /var/cache/cloudregister')
+            Path.create('/var/cache/cloudregister')
```

Comment:
The log message should now tell us where the bind mount is coming from i.e. should include `cloud_register_metadata`

Reply:
got it